contributing.md

```diff
@@ -395,19 +396,21 @@ Porting changes across branches:
   changes can be added in a backwards-compatible way (no API / ABI / behavior
   break in released branches).
 * Periodically, a maintainer will **forward-port** changes to newer release
-  branches all the way up to `master`.
-* The merge forward can be done with `git merge` in order to keep the commit history.
-  For example:
+  branches all the way up to `main`.
+* See [this list](https://github.com/ignitionrobotics/docs/blob/master/tools/branch_comparisons.md) to check if a branch needs porting.
+* The merge forward can be done with `git merge` in order to keep the commit history
+  and so it's easier to compare branches. For example:
 
         git checkout ign-<library>M
         git pull
         git checkout ign-<library>N
         git pull
-        git checkout -b M_to_N_<date> # It's important to do this before `git merge`
+        git checkout -b username/M_to_N_<date> # It's important to do this before `git merge`
         git merge ign-<library>M
         # Fix conflicts
         git commit -sam"Merge M into N"
         # Open pull request
+        # Do not squash or rebase, create a merge commit
 
 * In the rare event that a pull request needs to be backported (i.e. from a
   higher version to a lower version), use `git cherry-pick`, for example:
```

Comment:
```suggestion
* In the rare event that a pull request needs to be backported (i.e. from a
  higher version to a lower version), use `git cherry-pick` instead of `git merge`, for example:
```

I'm not sure if this is correct or not, but this is how I interpret it. If this suggestion is correct, I think it would be worth applying because the original backport instructions left me confused as to whether I should run `git merge` at all or not.

Reply:
824176b170a8353a14f69bde100af7982fd8aa53